src/core/dfn.js

```diff
@@ -32,3 +46,27 @@ export function run(conf) {
       }, $dfn);
   });
 }
+
+function autoPluralizeDfns() {
+  const links = [...document.querySelectorAll("a")];
+  const dfns = [...document.querySelectorAll("dfn")];
+
+  return function shouldPluralize(dfn) {
+    const plural = pluralize(norm(dfn.textContent));
+    if (
+      // has overridden to not do pluralization
+      dfn.hasAttribute("data-lt-noPlural") ||
```

Comment:
this should move up to the query for dfns (i.e., ":not([data-lt-noPlural])")

Reply:
not sure here. I'm doing it in one pass on all `dfn`.
If I add another pass, there might be those race conditions again. What do you think?

Reply:
It shouldn't, as noPlural is never added dynamically.